fetch.bs

```diff
@@ -2796,44 +2828,54 @@ steps:
  <dt>"<code>blob</code>"
  <dd>
   <ol>
-   <li><p>Let <var>blob</var> be <var>request</var>'s
-   <a for=request>current url</a>'s
-   <a>object</a>.
-
    <li>
-    <p>If <var>request</var>'s <a for=request>method</a> is not
-    `<code>GET</code>` or <var>blob</var> is null, then return a
-    <a>network error</a>.
+    <p>Run these substeps, but abort the substeps if fetch is <a for=fetch>terminated</a>, providing
+    a <var>reason</var>:
```

Comment:
This time the "providing" is up here, for unknown reasons... better to be consistent.